docs/gettingstarted/sc4snmp-installation.md

```diff
@@ -42,6 +42,22 @@ or download it directly from Helm using the command `microk8s helm3 show values
 
 It is recommended to start by completing the base template and gradually add additional configurations as needed.
 
+
+The `values.yaml` file is validated using `JSON schema` built into `helm chart` and additionally validated inside the code.
+If you want to make sure that your `values.yaml` follows formatting standards, you can use `yamllint`. In order to download
+`yamllint` refer to the [installation instructions](https://yamllint.readthedocs.io/en/stable/quickstart.html#installing-yamllint). 
+Then create `custom-config.yamllint` file and add the following configuration:
+```yaml
+extends: default
+
+rules:
+  line-length:
+    max: 80
+    level: warning
+
+```
```

Comment:
you can mention that it's also in our repo in examples or link it there

Reply:
Done